app/views/layouts/exercise_panel/_exercise.html.erb

```diff
@@ -6,6 +6,11 @@
     <%= exercise.submissions_count %>
   </td>
   <td>
+    <% if exercise.solved_by?(current_user) && current_user %>
+      <span class="glyphicon glyphicon-certificate passed-exercise" aria-hidden="true"></span>
+    <% elsif !exercise.solved_by?(current_user) && current_user %>
```

Comment:
It would be also nice to have a different color - grey ? -  for exercises never submitted by the user. Displaying them as their were failed -red - is a bit shocking I think.